src/neo/Network/P2P/Payloads/CosignerAttribute.cs

```diff
@@ -7,28 +7,30 @@
 
 namespace Neo.Network.P2P.Payloads
 {
-    public class Cosigner : ISerializable
+    public class CosignerAttribute : TransactionAttribute
     {
+        // This limits maximum number of AllowedContracts or AllowedGroups here
+        private const int MaxSubitems = 16;
```

Comment:
Now, we have 21 committees, 16 is not enough.

Reply:
32?

Reply:
What is the relationship between `MaxTransactionAttributes` and committee?

Reply:
Commitees need to send multisign transaction for setting policy parameters?

Reply:
But multisig doesn't use this field.

Reply:
You are right.